sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -49,7 +49,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-keys</artifactId>
-      <version>4.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-keys;current} -->
+      <version>4.3.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-security-keyvault-keys;current} -->
```

Comment:
We need to make sure the Cosmos team is aware of this dependency as it will become an issue for their next release as they will need to wait until keyvault releases.

Reply:
@kushagraThapar Similar to the Identity situation [here](https://github.com/Azure/azure-sdk-for-java/pull/15127#discussion_r487333250) cosmos could be taking a `dependency` tag unless we really need the source updates from KV for the azure-cosmos-encryption package.

Reply:
Made a small change at the end of this from `current` to `dependency`, else the pipelines would complain when verifying POM versions.